packages/app/src/specs/InlineSpecListHeader.vue

```diff
@@ -47,6 +47,18 @@
       >
         {{ t('specPage.searchPlaceholder') }}
       </label>
+      <button
+        v-if="props.search"
+        data-cy="clear-search-button"
+        class="border-transparent rounded-md flex outline-none h-24px my-4px inset-y-0 right-0 w-24px duration-300 absolute items-center justify-center group hocus-default hocus:ring-0"
+        :aria-label="t('specPage.clearSearch')"
+        @click.stop="clearInput"
+      >
+        <i-cy-delete_x16
+          class="icon-light-gray-1000 group-hocus:icon-dark-indigo-300"
+          :class="inputFocused ? 'icon-dark-indigo-300' : 'icon-dark-gray-800'"
+        />
+      </button>
     </div>
     <button
       tabindex="-1"
```

Comment:
Not part of this PR, but while we're in here, let's remove this `tabindex="-1"` that makes this New Spec button unreachable with the keyboard, I have no idea why it's there.